code/script.js

```diff
@@ -225,101 +233,197 @@ const generateBoard = () => {
 // Randomly select a person from the characters array and set as the value of the variable called secret
 const setSecret = () => {
   secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
+  console.log(`kukkuu`)
 }
 
 // This function to start (and restart) the game
 const start = () => {
   // Here we're setting charactersInPlay array to be all the characters to start with
   charactersInPlay = CHARACTERS
-  // What else should happen when we start the game?
+  generateBoard()
+  setSecret()
+  startTimer()
+  console.log(`hello`)
+  winOrLose.style.display='none'
+  guessCounter.innerHTML = `0`
```

Comment:
I think you could just write: `guessCounter = 0`; without backticks and then add
 gameGuess.innerHTML = `
${guessCounter}` 
if `gameGuess` is the varible for the questions asked :)